rdl/data_load_tracking/DataLoadExecution.py

```diff
@@ -23,3 +25,14 @@ class DataLoadExecution(Base):
     rows_processed = Column(Integer, nullable=False)
     model_checksum = Column(String(100), nullable=False)
     failure_reason = Column(String(1000), nullable=True)
+
+
+class Execution(Base):
+    __tablename__ = 'execution'
+    __table_args__ = {'schema': f'{Constants.DATA_PIPELINE_EXECUTION_SCHEMA_NAME}'}
+    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    execution_started = Column(DateTime(timezone=True), server_default=func.now())
+    execution_ended = Column(DateTime(timezone=True))
+    execution_time_ms = Column(Integer, nullable=True)
+    total_rows_processed = Column(Integer, nullable=True)
+    status = Column(String(25), nullable=False, default=Constants.ExecutionStatus.FAILED)
```

Comment:
also, let's split ExecutionStatus into:
- ExecutionStatus
- ModelExecutionStatus
- ModelBatchExecutionStatus
or something similar that diffirentiate's between an execution's, execution's model's and execution model's batch's statuses.

Reply:
It seemed like ExecutionStatus == ModelExecutionStatus so I've left them as one for now, split out the batch only statuses